packages/backend/src/workers/action.ts

```diff
@@ -124,18 +124,25 @@ worker.on('failed', async (job, err) => {
       job: job.data,
     },
   )
+
+  const flow = await Flow.query()
+    .findById(job.data.flowId)
+    .withGraphFetched('user')
+    .throwIfNotFound()
+
+  let shouldAlwaysSendEmail = false
+  if (flow.config?.errorConfig?.notificationFrequency === 'always') {
+    shouldAlwaysSendEmail = true
+  }
```

Comment:
nit: `const` might help prevent footguns, so I tend to like using it more over `let`.

```suggestion
  const shouldAlwaysSendEmail = flow.config?.errorConfig?.notificationFrequency === 'always')
```